examples/java/core/enumerations/src/main/bond/enumerations.bond

```diff
@@ -0,0 +1,21 @@
+namespace Examples;
+
+enum Color
+{
+    Orange = 1,
+    Yellow = 2,
+}
+
+enum Fruit
+{
+    Apple = 1,
+    Orange = 2,
+}
+
+enum Limits
+{
+    Int32Min = -2147483648,
+    Int32Max = 2147483647,
+    UInt32Min = 0,
+    UInt32Max = 0xFFFFFFFF,
```

Comment:
I'd really like to keep the examples the same across the different languages so that people can see the implications of some of their choices in various languages -- especially because many of our current customers who would be picking up Java have C# interop needs. We can add more documentation around this.